examples/sketch_inpaint.py

```diff
@@ -0,0 +1,31 @@
+from PIL import ImageFilter
+
+from multigen.pipes import MaskedIm2ImPipe
+import PIL.Image
+import numpy
+import torch
+import random
+
+
+def main():
+    blur = 8
+    model_dir = "./models-sd/"
+    model_id = "icb_diffusers_final"
```

Comment:
In other examples, "icbinp" is used. I'd propose to have the same naming of this model in all the examples, so they work simultaneously in the same settings.

Reply:
but inpainting model can't be used yet

Reply:
The model name is confusing. It's "I Can't Believe It's Not a Photography". It's not inpainting.